src/shipit/api/shipit_api/cli.py

```diff
@@ -233,6 +250,31 @@ def get_product_mozilla_version(product: Product,
 Products = typing.List[Product]
 
 
+def datetime_or_none(dt: typing.Optional[datetime.datetime]) -> typing.Optional[str]:
```

Comment:
you could even more generalize this into `or_none` and `to_isoformat`

```
A = typing.TypeVar('A')
B = typing.TypeVar('B')

def or_none(x: typing.Optional[A], func: typing.Callable[[A], B]) -> typing.Optional[B]:
    if x is None:
        return None
    return func(x)

def to_isoformat(d: datetime.datetime) -> str:
    return arrow.get(d).isoformat()
```

And then you can use it as `or_none(release.created, to_isoformat)` which can also support other Optional values